ci/badge/gen_badge.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+##
+# Copyright (C) 2019 Samsung Electronics
+# License: LGPL-2.1
```

Comment:
Do you want to use LGPL for this file? The TAOS-CI is officially released with Apache license.

Reply:
@leemgs  Changed to Apache-2.0.